cmd/botkube/main.go

```diff
@@ -55,6 +55,12 @@ func main() {
 		go mb.Start()
 	}
 
+	if Config.Communications.Discord.Enabled {
+		log.Logger.Info("Starting discord bot")
+		db := bot.NewDiscordBot()
```

Comment:
Can you please rename variable from `db` to `discordBot` ? Db confuses with database.

Reply:
expanded all short forms like sb,mb, and db respectively to maintain uniformity